lib/index.js

```diff
@@ -1,13 +1,26 @@
 var pkg = require('./pkg');
+var defaultInfo = require('./default-info');
 
 function info() {
-  var scripts = pkg()['scripts-info'];
+  var pkgJSON = pkg();
+  var scriptsInfo = pkgJSON['scripts-info'] || {};
 
-  if ( !scripts ) {
-    throw new Error('Oops, but no `scripts-info` property was found in your `package.json`');
+  Object.keys(pkgJSON.scripts || {})
+    .filter(function(scriptName) {
+      return !(scriptName in scriptsInfo);
+    })
+    .filter(function(scriptName) {
+      return scriptName in defaultInfo;
+    })
+    .map(function(scriptName) {
+      scriptsInfo[scriptName] = defaultInfo[scriptName];
+    });
```

Comment:
Extend would add all the default descriptions. For instance, it would add the description of `build` even if that script is not present in the `scripts` property